integration_test/cases/browser/touch_scroll_test.exs

```diff
@@ -0,0 +1,27 @@
+defmodule Wallaby.Integration.Browser.TouchScrollTest do
+  use Wallaby.Integration.SessionCase, async: true
+  alias Wallaby.Integration.Helpers
+
+  setup %{session: session} do
+    page = visit(session, "touch.html")
+
+    {:ok, %{page: page}}
+  end
+
+  describe "touch_scroll/4" do
+    test "scrolls the page using touch events", %{page: page} do
+      assert visible?(page, Query.text("Start", count: 0))
+      assert visible?(page, Query.text("Move", count: 0))
+      assert visible?(page, Query.text("End", count: 0))
```

Comment:
not sure how this works but shouldn't the above be `assert has?` with `count: 0` instead of `assert visible?` ?